server/core/store.go

```diff
@@ -103,15 +104,38 @@ func (s *StoreInfo) DownTime() time.Duration {
 }
 
 const minWeight = 1e-6
+const maxScore = 10000
+
+func (s *StoreInfo) score(size, highSpaceRatio, lowSpaceRatio float64) float64 {
+	capacity := float64(s.Stats.GetCapacity()) / (1 << 20)
+	available := float64(s.Stats.GetAvailable()) / (1 << 20)
+
+	// lowSpaceRatio shouldn't greater than highSpaceRatio
+	if lowSpaceRatio > highSpaceRatio {
+		lowSpaceRatio = highSpaceRatio
+	}
+
+	if available >= highSpaceRatio*capacity {
+		return size
+	}
+
+	if available <= lowSpaceRatio*capacity {
+		return maxScore - available
+	}
+
+	k := (maxScore - (lowSpaceRatio+highSpaceRatio)*capacity) / ((1 - lowSpaceRatio - highSpaceRatio) * capacity)
```

Comment:
add some description about the formula

Reply:
Does it work well when the capacity is 1T?